7_react/react1/week3/todo-list-app/src/EmployeesList.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+
+const Employees = () => {
+  const [data, setData] = useState([]);
+
+  const fetchData = async ()=>{
+
+    try {
+        let res = await fetch('https://gist.githubusercontent.com/benna100/5fd674171ea528d7cd1d504e9bb0ca6f/raw')
```

Comment:
Since `res` and `data` are not _modified_ later after being set, you can just use `const`. 
Of course it works this way too, but differentiating between `const` and `let` helps make the code easier to read.